publishPrReleaseAndCreateFreshCraTemplate.js

```diff
@@ -41,7 +41,7 @@ runExternalCommandSync(
     '--scripts-version',
     `@fs/react-scripts@${newVersion}`,
     '--template',
-    '@fs/cra-template@next',
+    '@fs/cra-template',
```

Comment:
Pointing to next on the frontierMaster branch is what published fresh-cra-template back onto v12.